discovery-provider/src/tasks/users.py

```diff
@@ -79,6 +77,83 @@ def user_state_update(
                             txhash,
                         )
                         user_events_lookup[user_id] = {"user": ret_user, "events": []}
+                except Exception as e:
+                    logger.error("Error in parse user transaction")
+                    event_blockhash = update_task.web3.toHex(block_hash)
+                    raise IndexingError(
+                        "user", block_number, event_blockhash, txhash, str(e)
+                    ) from e
+
+    creator_node_endpoint = {
```

Comment:
why do we need this?

Reply:
@cheran-senthil - to clarify the above q is this to limit which endpoints we query for this particular CID? regardless i think this variable should be clearer